### ISSUE TITLE
Status of project?

### ISSUE THREAD

**Issue by ffleandro** [moshbit/Kotlift#13]
What is the current status of this project?
Please update, since the official support for Kotlin on Android, this project is a must need.

**Comment by vonox7** [comment]
Currently this project isn't under active maintenance. Feel free to fix any issues and creating pull requests.
I personally hope, that with Kotlin/Native this approach isn't needed any more.

**Comment by formvoltron** [comment]
So with Kotlin/Native, I could use Kotlin to write some code that is can be shared between an iOS and Android project?  Then write the UI code in each native environment?

Now I use Java & J2Objc, but wanting to modernize.

**Comment by vonox7** [comment]
Yes, thats true. You can read more on https://blog.jetbrains.com/kotlin/2017/12/kotlinnative-v0-5-released-calling-kotlin-from-swift-and-c-llvm-5-and-more/ .

**Comment by formvoltron** [comment]
Thank you for the link.  Ideally I hope to use Flutter for UI and Kotlin for platform independent logic.  Or maybe it would be sufficient to code the logic in Dart (Flutter's language).
